core/controllers/profile.py

```diff
@@ -133,6 +132,16 @@ def get(self):
         })
         self.render_json(self.values)
 
+class ProfilePictureHandlerByUsername(base.BaseHandler):
+    """ Provides the dataURI of the profile picture of the specified user,
+    or none if no user picture is uploaded for the user with that ID."""
```

Comment:
Hey Sean, all good on the raising the PageNotFoundException, I think (have yet to push this change though). Could you elaborate on what you mean by "in docstring s/none/None"? Thanks!